DataGridSample/DataGridSample/ViewModels/MainViewModel.cs

```diff
@@ -45,6 +45,14 @@ public bool IsRefreshing
 		public ICommand RefreshCommand { get; set; }
 		#endregion
 
+
+		public void GridDoubleTapped(ItemTappedEventArgs e)
```

Comment:
Convenient way to handle a command in ViewModel should be ICommand instead event. You might remove this method or add ICommand to DataGrid.